gtsfm/retriever/joint_netvlad_sequential_retriever.py

```diff
@@ -0,0 +1,60 @@
+"""Retriever that includes both sequential and retrieval links.
+
+Authors: John Lambert
+"""
+
+from typing import List, Tuple
+
+import dask
+
+import gtsfm.utils.logger as logger_utils
+from gtsfm.loader.loader_base import LoaderBase
+from gtsfm.retriever.retriever_base import RetrieverBase
+from gtsfm.retriever.netvlad_retriever import NetVLADRetriever
+from gtsfm.retriever.sequential_retriever import SequentialRetriever
+
+
+logger = logger_utils.get_logger()
+
+
+class JointNetVLADSequentialRetriever(RetrieverBase):
+    """Note: this class contains no .run() method."""
```

Comment:
missing class documentation

Reply:
Do we have a standard for this? I prefer the class documentation to be more verbose, because it will show up in IDEs (if you hover over an instance of the class) and other doc tools we may want to use in the future. If not more verbose at least somewhat descriptive.

Reply:
Hmm didn't realize that, I use Sublime Text : - )

Happy to put more info in the class docstring here, though.

Reply:
Yeah, it would be nice if you could add a short summary now, but for future reference I think class and method documentation should be preferred over file documentation. In GTSAM for example, we just have the copyright and a brief one or two liner for the file.

Reply:
Unresolved :)